### PR TITLE
feat: add L3norm embedding evaluator

### PR DESCRIPTION
**Pull request by smy0428**
Split the large PR into small ones.

Add L3 norm (Minkowski distance where p=3) for embedding evaluator.
I know it is a bit pointless to write L3 norm than to L1, L2 or L infinity, but all the other ordinary norms have been written. Minkowski distance can also be extended to any p >= 1, just more math problems.

### CODE REVIEW

File: evaluators/embedding/L3NormEvaluator/__init__.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class L3NormEvaluator(BaseEmbeddingEvaluator):
+    """A :class:`L3NormEvaluator` evaluates the distance between actual and desired embeddings computing
+    the Minkowski distance ( p=3 ) between them.
```
Comment (JoanFM) [inline evaluators/embedding/L3NormEvaluator/__init__.py:8]: is there a way to parametrize the `p`?

**Comment by JoanFM** [comment]
Thanks for your contribution. Unfortunately, as you didn't set up your local git config correctly, the author of your commit does not match with your Github account. This poses problems:
Your contribution can not be credited, and will not show up in the list of contributors (as bad git config = anonymous)
We can not accept CLA for anonymous contribution.
Please note that this does not only affect your contribution to Jina, but also affects all your contribution to anything on Github. So better fix it, once for all.
Here is what you can do:
```
git config user.name "Correct new name"
git config user.email "correct@new.email" -> match with your Github registered email address
```
Then fix the commit message by
```
git commit --amend --author="Author Name <email@address.com>" --no-edit
git push --force
```

File: evaluators/embedding/L3NormEvaluator/__init__.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class L3NormEvaluator(BaseEmbeddingEvaluator):
```
Comment (JoanFM) [inline evaluators/embedding/L3NormEvaluator/__init__.py:6]: Let's have it parametrizable by p.

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: evaluators/embedding/L3NormEvaluator/__init__.py (hunk 2 above)
Comment (smy0428) [reply evaluators/embedding/L3NormEvaluator/__init__.py:6]: This L3norm evaluator could be replaced by the more general Hamming-distance-evaluator

**Comment by JoanFM** [comment]
Closing this as it can be done in MinkowskiDistanceEvaluator

**Pull request closed without merging**